### PR TITLE
dcmio: dcmanonymizer: adding anonimyzer pipeline. With pilots

### PR DESCRIPTION
**Pull request by dgoyard**
dcmio: dcmanonymizer: adding anonimyzer pipeline. With pilots
dcmio: dcmreader: adding dicom reader functions (utils)

### CODE REVIEW

File: dcmio/dcmreader/dcmreader.py [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+    return echo time
+    """
+    dataset = dicom.read_file(path_to_dicom, force=True)
+    value = walk(dataset, walker_callback, (0x0018, 0x0081))
+    if value:
+        return value
+    return -1
+
+
+def get_SOP_storage_type(path_to_dicom):
```
Comment (AGrigis) [inline dcmio/dcmreader/dcmreader.py]: sop

File: dcmio/dcmreader/dcmreader.py [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+    """
+    dataset = dicom.read_file(path_to_dicom, force=True)
+    value = walk(dataset, walker_callback, (0x0008, 0x0016))
+    if value:
+        if "Enhanced" in str(value):
+            return True
+    return False
+
+
+def get_Raw_Data_Run_Number(path_to_dicom):
```
Comment (AGrigis) [inline dcmio/dcmreader/dcmreader.py]: raw_data_run_number

**Comment by AGrigis** [comment]
-> remove *.pyc and *.py~ compiled/temp files.
-> add .gitignore.
->uniformize docstring and add 'Parameters' and 'Returns' descriptions.

**Comment by dgoyard** [comment]
-> remove *.pyc and *.py~ compiled/temp files.                                                        OK
-> add .gitignore.                                                                                                       OK
->uniformize docstring and add 'Parameters' and 'Returns' descriptions.                 OK

**Pull request merged**